usr/sbin/pkcsslotd/garbage_linux.c

```diff
@@ -493,7 +493,7 @@ BOOL IsValidProcessEntry(pid_t_64 pid, time_t_64 RegTime)
         Err = errno;
         if (Err == ESRCH) {
             /* The process was not found */
-            DbgLog(DL3, "IsValidProcessEntry: PID %d was not found in the "
+            DbgLog(DL3, "IsValidProcessEntry: PID %llu was not found in the "
                    "process table (kill() returned %s)",
```

Comment:
I saw that pid_t_64 is a 'signed long long' (see slotmgr.h). Why not %lld?

Reply:
Ah yes, you are correct. Will change to `lld` then.